### PR TITLE
Add ep.32

### PR DESCRIPTION
**Pull request by PercyDan54**

### CODE REVIEW

File: S3/English/ASS_32.ass [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Dialogue: 0,0:01:30.81,0:01:31.33,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:32.42,0:01:33.69,Default,,0,0,0,,So hard to decide
+Dialogue: 0,0:01:33.97,0:01:34.79,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:35.25,0:01:36.11,Default,,0,0,0,,We're next
+Dialogue: 0,0:01:37.97,0:01:38.90,Default,,0,0,0,,Let's go
+Dialogue: 0,0:01:42.04,0:01:43.64,Default,,0,0,0,,Next
+Dialogue: 0,0:01:54.39,0:01:55.09,Default,,0,0,0,,It's awesome!
+Dialogue: 0,0:01:55.26,0:01:56.75,Default,,0,0,0,,Let's do this again. Luis
+Dialogue: 0,0:01:58.39,0:01:58.80,Default,,0,0,0,,Lulu
+Dialogue: 0,0:01:59.64,0:02:01.60,Default,,0,0,0,,Somebody help! Somebody died
```
Comment (PercyDan54) [inline S3/English/ASS_32.ass:46]: Need better tl

File: S3/English/ASS_32.ass [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Dialogue: 0,0:02:44.80,0:02:46.14,Default,,0,0,0,,I couldn't learn it
+Dialogue: 0,0:02:47.38,0:02:48.55,Default,,0,0,0,,I'll teach you
+Dialogue: 0,0:02:48.67,0:02:49.99,Default,,0,0,0,,Really? I will try my best
+Dialogue: 0,0:02:51.61,0:02:52.21,Default,,0,0,0,,But
+Dialogue: 0,0:02:52.78,0:02:54.44,Default,,0,0,0,,If I 
+Dialogue: 0,0:02:54.44,0:02:55.91,Default,,0,0,0,,
+Dialogue: 0,0:02:56.40,0:02:57.78,Default,,0,0,0,,No problem, I'm on it
+Dialogue: 0,0:03:00.45,0:03:01.80,Default,,0,0,0,,Listen to my instructions!
+Dialogue: 0,0:03:02.04,0:03:04.31,Default,,0,0,0,,
+Dialogue: 0,0:03:04.31,0:03:06.08,Default,,0,0,0,,And
```
Comment (PercyDan54) [inline S3/English/ASS_32.ass:64]: Need better tl

File: S3/English/ASS_32.ass [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Dialogue: 0,0:02:42.23,0:02:42.84,Default,,0,0,0,,Lily
+Dialogue: 0,0:02:44.90,0:02:46.00,Default,,0,0,0,,I couldn't learn it
+Dialogue: 0,0:02:47.38,0:02:48.35,Default,,0,0,0,,I'll teach you
+Dialogue: 0,0:02:48.67,0:02:49.99,Default,,0,0,0,,Really? I will try my best
+Dialogue: 0,0:02:51.61,0:02:52.15,Default,,0,0,0,,But
+Dialogue: 0,0:02:52.78,0:02:54.20,Default,,0,0,0,,If I'm in the same pool as you
+Dialogue: 0,0:02:54.52,0:02:55.80,Default,,0,0,0,,I'll turn into a goat
+Dialogue: 0,0:02:56.40,0:02:57.78,Default,,0,0,0,,No problem, I'm on it
+Dialogue: 0,0:03:00.45,0:03:01.80,Default,,0,0,0,,Listen to my instructions!
+Dialogue: 0,0:03:02.15,0:03:04.20,Default,,0,0,0,,1
```
Comment (Hrshl7) [inline S3/English/ASS_32.ass:67]: isn't that where Lily is screaming "instructions"? 
"Your feet/legs are kicking and kicking, your hands/arms are sculling and sculling" or something?
then she's just yelling 1, 2, 1, 2

File: S3/English/ASS_32.ass [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Dialogue: 0,0:03:57.90,0:03:59.20,Default,,0,0,0,,Just deceptive tricks
+Dialogue: 0,0:03:59.55,0:04:00.47,Default,,0,0,0,,Why would I be shy
+Dialogue: 0,0:04:00.98,0:04:01.88,Default,,0,0,0,,Who was deceptive
+Dialogue: 0,0:04:02.24,0:04:03.28,Default,,0,0,0,,Those are real
+Dialogue: 0,0:04:03.52,0:04:04.28,Default,,0,0,0,,Touch it if you don't belive me
+Dialogue: 0,0:04:05.21,0:04:06.00,Default,,0,0,0,,Pervert
+Dialogue: 0,0:04:06.60,0:04:08.00,Default,,0,0,0,,Why did you hurt me
+Dialogue: 0,0:04:14.38,0:04:15.64,Default,,0,0,0,,Abu
+Dialogue: 0,0:04:15.64,0:04:16.62,Default,,0,0,0,,We're leaving
+Dialogue: 0,0:04:17.18,0:04:18.44,Default,,0,0,0,,How long had he stay there
```
Comment (Hrshl7) [inline S3/English/ASS_32.ass:107]: How long has he been here?

File: S3/English/ASS_32.ass [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+Dialogue: 0,0:03:50.65,0:03:52.61,Default,,0,0,0,,Transformation is an optional course in Devil University
+Dialogue: 0,0:03:52.94,0:03:54.81,Default,,0,0,0,,And Neil couldn't become such a beauty like me
+Dialogue: 0,0:03:55.35,0:03:56.55,Default,,0,0,0,,Are you shy?
+Dialogue: 0,0:03:57.90,0:03:59.20,Default,,0,0,0,,Just deceptive tricks
+Dialogue: 0,0:03:59.55,0:04:00.47,Default,,0,0,0,,Why would I be shy
+Dialogue: 0,0:04:00.98,0:04:01.88,Default,,0,0,0,,Who was deceptive
+Dialogue: 0,0:04:02.24,0:04:03.28,Default,,0,0,0,,Those are real
+Dialogue: 0,0:04:03.52,0:04:04.28,Default,,0,0,0,,Touch it if you don't belive me
+Dialogue: 0,0:04:05.21,0:04:06.00,Default,,0,0,0,,Pervert
+Dialogue: 0,0:04:06.60,0:04:08.00,Default,,0,0,0,,Why did you hurt me
```
Comment (Hrshl7) [inline S3/English/ASS_32.ass:104]: "slap" would be better since the wound is on his face?

File: S3/English/ASS_32.ass [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+Dialogue: 0,0:03:47.25,0:03:48.00,Default,,0,0,0,,It's me
+Dialogue: 0,0:03:48.24,0:03:49.20,Default,,0,0,0,,Surprised?
+Dialogue: 0,0:03:50.65,0:03:52.61,Default,,0,0,0,,Transformation is an optional course in Devil University
+Dialogue: 0,0:03:52.94,0:03:54.81,Default,,0,0,0,,And Neil couldn't become such a beauty like me
+Dialogue: 0,0:03:55.35,0:03:56.55,Default,,0,0,0,,Are you shy?
+Dialogue: 0,0:03:57.90,0:03:59.20,Default,,0,0,0,,Just deceptive tricks
+Dialogue: 0,0:03:59.55,0:04:00.47,Default,,0,0,0,,Why would I be shy
+Dialogue: 0,0:04:00.98,0:04:01.88,Default,,0,0,0,,Who was deceptive
+Dialogue: 0,0:04:02.24,0:04:03.28,Default,,0,0,0,,Those are real
+Dialogue: 0,0:04:03.52,0:04:04.28,Default,,0,0,0,,Touch it if you don't belive me
```
Comment (Hrshl7) [inline S3/English/ASS_32.ass:102]: believe

File: S3/English/ASS_32.ass [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+Dialogue: 0,0:04:05.21,0:04:06.00,Default,,0,0,0,,Pervert
+Dialogue: 0,0:04:06.60,0:04:08.00,Default,,0,0,0,,Why did you hurt me
+Dialogue: 0,0:04:14.38,0:04:15.64,Default,,0,0,0,,Abu
+Dialogue: 0,0:04:15.64,0:04:16.62,Default,,0,0,0,,We're leaving
+Dialogue: 0,0:04:17.18,0:04:18.44,Default,,0,0,0,,How long had he stay there
+Dialogue: 0,0:04:18.97,0:04:20.09,Default,,0,0,0,,Won't he become soak?
+Dialogue: 0,0:04:22.53,0:04:24.03,Default,,0,0,0,,But seems he really liked bathing
```
Comment (Hrshl7) [inline S3/English/ASS_32.ass:109]: "but it seems he really likes bathing"
"liked" work too but he's still bathing so I personally go with that being a constant fact than a one time one

File: S3/English/ASS_32.ass (hunk 1 above)
Comment (Hrshl7) [reply S3/English/ASS_32.ass:49]: it actually fits the situation so maybe ask Tori instead.

File: S3/English/ASS_32.ass [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+Dialogue: 0,0:02:01.76,0:02:02.84,Default,,0,0,0,,Luis
+Dialogue: 0,0:02:11.63,0:02:12.67,Default,,0,0,0,,Lynn
+Dialogue: 0,0:02:13.44,0:02:15.67,Default,,0,0,0,,Could you go to another pool for a moment
+Dialogue: 0,0:02:15.90,0:02:16.40,Default,,0,0,0,,What's the matter
+Dialogue: 0,0:02:16.90,0:02:19.60,Default,,0,0,0,,You and Lily had turned all the water in the pool into holy water
+Dialogue: 0,0:02:20.70,0:02:21.90,Default,,0,0,0,,So where should we go then
+Dialogue: 0,0:02:24.64,0:02:25.90,Default,,0,0,0,,I got an idea, come with me
+Dialogue: 0,0:02:40.19,0:02:41.50,Default,,0,0,0,,Can't you swim, Neil
+Dialogue: 0,0:02:42.23,0:02:42.84,Default,,0,0,0,,Lily
+Dialogue: 0,0:02:44.90,0:02:46.00,Default,,0,0,0,,I couldn't learn it
```
Comment (Hrshl7) [inline S3/English/ASS_32.ass:59]: "I never learned" would be better imo

File: S3/English/ASS_32.ass [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+Dialogue: 0,0:02:51.61,0:02:52.15,Default,,0,0,0,,But
+Dialogue: 0,0:02:52.78,0:02:54.20,Default,,0,0,0,,If I'm in the same pool as you
+Dialogue: 0,0:02:54.52,0:02:55.80,Default,,0,0,0,,I'll turn into a goat
+Dialogue: 0,0:02:56.40,0:02:57.78,Default,,0,0,0,,No problem, I'm on it
+Dialogue: 0,0:03:00.45,0:03:01.80,Default,,0,0,0,,Listen to my instructions!
+Dialogue: 0,0:03:02.15,0:03:04.20,Default,,0,0,0,,1
+Dialogue: 0,0:03:04.25,0:03:06.08,Default,,0,0,0,,2
+Dialogue: 0,0:03:04.22,0:03:04.65,Default,,0,0,0,,{\pos(970,200)}Can't...
+Dialogue: 0,0:03:05.05,0:03:06.00,Default,,0,0,0,,{\pos(970,200)}Understand it
+Dialogue: 0,0:03:08.03,0:03:08.25,Default,,0,0,0,,{\c&H548AE5&\fnTrebuchet MS\pos(1635,805)}Being a part-time lifeguard
```
Comment (Hrshl7) [inline S3/English/ASS_32.ass:71]: just "Part-time lifeguard" works

File: S3/English/ASS_32.ass (hunk 8 above)
Comment (PercyDan54) [reply S3/English/ASS_32.ass:59]: No. He meant he did actually tried but failed to learn

File: S3/English/ASS_32.ass (hunk 6 above)
Comment (PercyDan54) [reply S3/English/ASS_32.ass:102]: How did this pass the spell check

**Pull request merged**